### PR TITLE
build(nodejs): add support for node18

### PR DESCRIPTION
**Pull request by Engerim**
aws announced support for nodejs 18, to allow us the upgrade we need to have a nodejs 18 layer

### CODE REVIEW

**Review by akshaydk — APPROVED** [review]

**Comment by mrickard** [comment]
@Engerim Thank you for contributing! We're reviewing this, and we'll keep this issue updated.

File: dockerfiles/Dockerfile.nodejs18x [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN useradd -m newrelic-lambda-layers
+USER newrelic-lambda-layers
+WORKDIR /home/newrelic-lambda-layers
+
+COPY --chown=newrelic-lambda-layers libBuild.sh .
+COPY --chown=newrelic-lambda-layers nodejs nodejs/
+
+WORKDIR nodejs
+RUN ./publish-layers.sh build-nodejs16x
```
Comment (mrickard) [inline dockerfiles/Dockerfile.nodejs18x:14]: This should be 
`RUN ./publish-layers.sh build-nodejs16x`

File: dockerfiles/Dockerfile.nodejs18x [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+WORKDIR /home/newrelic-lambda-layers
+RUN pip3 install -U awscli --user
+ENV PATH /home/newrelic-lambda-layers/.local/bin/:$PATH
+
+COPY libBuild.sh .
+COPY nodejs nodejs/
+COPY --from=builder /home/newrelic-lambda-layers/nodejs/dist nodejs/dist
+
+WORKDIR nodejs
+CMD ./publish-layers.sh publish-nodejs16x
```
Comment (mrickard) [inline dockerfiles/Dockerfile.nodejs18x:29]: This should be 
`RUN ./publish-layers.sh build-nodejs18x`

**Review by mrickard — CHANGES_REQUESTED** [review]
Thank you! Just a couple fixes to the Dockerfile, and we should be good to go.

**Comment by Engerim** [comment]
@mrickard I have adjusted the two values

**Review by mrickard — APPROVED** [review]
Thank you, @Engerim !

**Comment by mrickard** [comment]
@Engerim I can take care of the conflicts--I'll just clone your fork.

**Comment by mrickard** [comment]
@Engerim Thank you again! I've incorporated your changes in this PR, with the conflicts fixed: https://github.com/newrelic/newrelic-lambda-layers/pull/122 (We're removing support for the deprecated Python 3.6 runtime also.)

**Pull request closed without merging**